tasks/minsize/minsize.snap

```diff
@@ -1,26 +1,26 @@
 Original   | Minified   | esbuild    | Gzip       | esbuild   
 
-72.14 kB   | 24.46 kB   | 23.70 kB   | 8.65 kB    | 8.54 kB    | react.development.js
+72.14 kB   | 24.12 kB   | 23.70 kB   | 8.62 kB    | 8.54 kB    | react.development.js
 
-173.90 kB  | 61.68 kB   | 59.82 kB   | 19.53 kB   | 19.33 kB   | moment.js 
+173.90 kB  | 61.67 kB   | 59.82 kB   | 19.53 kB   | 19.33 kB   | moment.js 
 
-287.63 kB  | 92.83 kB   | 90.07 kB   | 32.29 kB   | 31.95 kB   | jquery.js 
+287.63 kB  | 92.70 kB   | 90.07 kB   | 32.27 kB   | 31.95 kB   | jquery.js 
 
-342.15 kB  | 124.11 kB  | 118.14 kB  | 44.80 kB   | 44.37 kB   | vue.js    
+342.15 kB  | 121.95 kB  | 118.14 kB  | 44.59 kB   | 44.37 kB   | vue.js    
```

Comment:
Interesting that minified output is larger?

Reply:
Isn't this smaller? 🤔